src/cookie-consent.mjs

```diff
@@ -214,7 +210,7 @@ export default class Dialog extends HTMLElement {
   }
 
   initDomToggler() {
-    return DomToggler(Config());
+    return DomToggler(this.cookies);
```

Comment:
the `DomToggler` was called with the wrong property. `Config()` doesn't actually return infomartion about the current state of cookies, only initial configuration data. Therefor, the Domtoggler couldn't show or hide elements based on consent, since there was no info about consent.